backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/HybridBackupCommand.java

```diff
@@ -237,6 +237,31 @@ private void auditLog(AuditLogType type) {
         auditLogDirector.log(this, type);
     }
 
+    @Override
+    public AuditLogType getAuditLogTypeValue() {
+        VmBackup vmBackup = getParameters().getVmBackup();
+        addCustomValue("VmName", getVm().getName());
```

Comment:
I think that's not needed since this is a VmCommand (but I see it was there already so just something to consider only if there would be other changes..)